spyder/widgets/tests/test_status.py

```diff
@@ -67,74 +67,74 @@ def test_status_bar_widget_signal(status_bar, qtbot):
 
 
 def test_status_bar_conda_interpreter_status(status_bar, qtbot, mocker):
```

Comment:
The other tests (i.e. `test_status_bar_pyenv_interpreter_status` and `test_status_bar_internal_interpreter_status`) can be left as they are because we don't a way to test things directly with them.